### PR TITLE
PP-1816 Vuls reports aws compliance

### PR DESCRIPTION
**Pull request by joshmyers**
#### WHAT

See commits. This PR adds Vuls report scanning to the aws_compliance Lambda function. See updated README for example.

### CODE REVIEW

File: aws_compliance.py [hunk 1]
```diff
@@ -91,6 +98,114 @@ def should_skip_bucket(bucket):
...
+       summary[cve_id]['severity'] = cve_severity(cve)
+   summary[cve_id]['instances'] = summary[cve_id].get('instances') or []
+   summary[cve_id]['instances'].append(servername)
+
+
+def cve_score(cve):
+    if cve['CveDetail']['Nvd']['Score'] > 0:
+        return cve['CveDetail']['Nvd']['Score']
+    elif cve['CveDetail']['Jvn']['Score'] > 0:
+        return cve['CveDetail']['Nvd']['Score']
```
Comment (bazbremner) [inline aws_compliance.py]: The condition is looking at the JVN score, but setting NVD in the return value. It looks wrong; if it isn't, please add a comment to the code as to why this is this way for future generations.

**Review by bazbremner — CHANGES_REQUESTED** [review]
I can't see anything obviously amiss on review with the exception of the JVN/NVD item commented separately.

The 22 line conditional block handling `response['Contents']` is fairly dense.

File: aws_compliance.py (hunk 1 above)
Comment (joshmyers) [reply aws_compliance.py]: Good spot, amended.

**Comment by joshmyers** [comment]
@bazbremner comment about Jvd addressed, pushed and rebased.

**Review by bazbremner — APPROVED** [review]

**Pull request merged**